test/api/clientTest.js

```diff
@@ -11,7 +11,7 @@ describe('fetchCurrentUserPermissions', () => {
     const response = apiClient.fetchCurrentUserPermissions()
 
     expect(response).to.eq(null)
-    expect(stub).to.have.been.calledWith('POST', san.SANTIMENT_GRAPHQL_URL, {
```

Comment:
Use the constant here

Reply:
I actually brought the var back, because I can't import constants through the files